fastlane/Fastfile

```diff
@@ -17,4 +17,21 @@ platform :ios do
       reviewers: [:gersonnoboa, :dukhovnyi, :igorkravchenko, :EgorovEI]
     )
   end
+
```

Comment:
just an idea, but... do we need to update template for Widgets SDK `fastlane/templates/GliaWidgets.podspec` after Core SDK release and replace
```
# from
s.dependency 'SalemoveSDK', '#{prev_version}
# to
s.dependency 'SalemoveSDK', '#{version}'
```

Reply:
Yes, that is done [here](https://github.com/salemove/glia-ios-podspecs/pull/50/files#diff-c39d944cdc46aa43c8495a4631fd7c8ebff578992b9df81e358bca391b375acbR11).